x-pack/plugin/security/cli/build.gradle

```diff
@@ -0,0 +1,22 @@
+apply plugin: 'elasticsearch.build'
+
+archivesBaseName = 'elasticsearch-security-cli'
+dependencies {
+    compileOnly "org.elasticsearch:elasticsearch:${version}"
+    compile 'org.bouncycastle:bcprov-jdk15on:1.59'
+    compile 'org.bouncycastle:bcpkix-jdk15on:1.59'
+}
+
+jar {
+    from(project(':x-pack:plugin:core').sourceSets.main.output.classesDirs) {
```

Comment:
can we not move the source files to this project? At the very least I think we should exclude these from the x-pack core jar

Reply:
Can you elaborate please? We're not moving source files to this project now. Am i reading your comment wrong or maybe the _not_ in _can we not move_ isn't meant to be there ?

I - probably falsely - assumed we do not want to move the source files to this project since we preferred this approach to the new plugin approach I tried out yesterday. It definitely makes sense to not have the classes both in this jar and in x-pack-core though - thanks.

Reply:
Sorry, my comment was not clear. I'd like for us to move the source files to this project for a clearer separation.

The distinction between a separate plugin vs a jar is that the certificate generation code doesn't require a plugin and just needs to be packaged in a way that it can be loaded as a CLI tool. I suggested this way as an alternative to a completely separate plugin to avoid any confusion about seeing a `security-cli` plugin/module deployed.

Reply:
Great thanks, all clear now. I'll move the source files in this project